ui/pages/create-account/import-account/import-account.js

```diff
@@ -0,0 +1,159 @@
+import React, { useContext, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { EVENT, EVENT_NAMES } from '../../../../shared/constants/metametrics';
+import { ButtonLink, Label, Text } from '../../../components/component-library';
+import Box from '../../../components/ui/box';
+import Dropdown from '../../../components/ui/dropdown';
+import { MetaMetricsContext } from '../../../contexts/metametrics';
+import {
+  BLOCK_SIZES,
+  BorderColor,
+  FONT_WEIGHT,
+  JustifyContent,
+  Size,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import ZENDESK_URLS from '../../../helpers/constants/zendesk-url';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { useRouting } from '../../../hooks/useRouting';
+import * as actions from '../../../store/actions';
+
+// Subviews
+import JsonImportView from './json';
+import PrivateKeyImportView from './private-key';
+
+export default function NewAccountImportForm() {
+  const t = useI18nContext();
+  const dispatch = useDispatch();
+  const trackEvent = useContext(MetaMetricsContext);
+  const { navigateToMostRecentOverviewPage } = useRouting();
+
+  const menuItems = [t('privateKey'), t('jsonFile')];
+
+  const [type, setType] = useState(menuItems[0]);
+
+  function importAccount(strategy, importArgs) {
+    const loadingMessage = getLoadingMessage(strategy);
+
+    dispatch(actions.importNewAccount(strategy, importArgs, loadingMessage))
+      .then(({ selectedAddress }) => {
+        if (selectedAddress) {
+          trackImportEvent(strategy, true);
+          dispatch(actions.hideWarning());
+          navigateToMostRecentOverviewPage();
+        } else {
+          dispatch(actions.displayWarning(t('importAccountError')));
+          trackImportEvent(strategy, false);
+        }
+      })
+      .catch((error) => translateWarning(error.message));
+  }
+
+  function trackImportEvent(strategy, wasSuccessful) {
+    const accountImportType =
+      strategy === 'Private Key'
+        ? EVENT.ACCOUNT_IMPORT_TYPES.PRIVATE_KEY
+        : EVENT.ACCOUNT_IMPORT_TYPES.JSON;
+
+    const event = wasSuccessful
+      ? EVENT_NAMES.ACCOUNT_ADDED
+      : EVENT_NAMES.ACCOUNT_ADD_FAILED;
+
+    trackEvent({
+      category: EVENT.CATEGORIES.ACCOUNTS,
+      event,
+      properties: {
+        account_type: EVENT.ACCOUNT_TYPES.IMPORTED,
+        account_import_type: accountImportType,
+      },
+    });
+  }
+
+  function getLoadingMessage(strategy) {
+    if (strategy === 'JSON File') {
+      return (
+        <Text width={BLOCK_SIZES.THREE_FOURTHS} fontWeight={FONT_WEIGHT.BOLD}>
+          <br />
+          {t('importAccountJsonLoading1')}
+          <br />
+          <br />
+          {t('importAccountJsonLoading2')}
+        </Text>
```

Comment:
I'm not a huge fan of using `<br />` tags to space out text. But understand that there is some performance benefits to using less complex jsx instead of using 2 `Text` components